db/data/20230626113331_sync_trainees_to_dqt_prior_to_feb2023.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class SyncTraineesToDqtPriorToFeb2023 < ActiveRecord::Migration[7.0]
+  def up
+    # Find all trainees in training or deferred where they have not been updated since Feb 2023
+    trainees = Trainee.where.not(trn: nil).where("length(trn) = 7").in_training.or(Trainee.deferred).where("updated_at <= ?", Date.new(2023, 2).end_of_month)
```

Comment:
I think this will mean all `deferred` trainees will be included in the set regardless of their `updated_at` although I could be mis-interpreting 🤔

Reply:
I think if using the `.or` we need to include the `trn` and `updated_at` constraints again within it

Reply:
good spot, i've split these out